src/ansys/pyensight/core/utils/export.py

```diff
@@ -464,3 +462,100 @@ def _animation_remote(
                 mp4_data = fp.read()
 
         return mp4_data
+
+    GEOM_EXPORT_GLTF = "gltf2"
+    GEOM_EXPORT_AVZ = "avz"
+    GEOM_EXPORT_PLY = "ply"
+    GEOM_EXPORT_STL = "stl"
+
+    extension_map = {
+        GEOM_EXPORT_GLTF: ".glb",
+        GEOM_EXPORT_AVZ: ".avz",
+        GEOM_EXPORT_PLY: ".ply",
+        GEOM_EXPORT_STL: ".stl",
+    }
+
+    def _geometry_remote(
+        self, format: str, begin_timestep: int = 0, end_timestep: int = 0, delta_timestep: int = 1
+    ) -> bytes:
+        """EnSight-side implementation.
+
+        Parameters
+        ----------
+        format : str
+            The format to export
+        begin_timestep: int
+            The first timestep to export
+        end_timestep: int
+            The final timestep to export
+        delta_timestep: int
+            The delta timestep to use when exporting
+
+        Returns
+        -------
+        bytes
+            Geometry export in bytes
+        """
+        rawdata = None
+        extension = self.extension_map.get(format)
+        if not extension:
+            raise RuntimeError("The geometry export format provided is not supported.")
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            self._ensight.objs.core.PARTS.set_attr("SELECTED", True)
```

Comment:
This is the UI notion of selection, not command language (you are using savegeom() which is command language).  

self._ensight.part.select_all()

is more correct here.

Reply:
uhm you're right, let me fix it